x-pack/plugins/monitoring/server/plugin.ts

```diff
@@ -189,9 +189,14 @@ export class MonitoringPlugin
     const config = createConfig(this.initializerContext.config.get<TypeOf<typeof configSchema>>());
     // Start our license service which will ensure
     // the appropriate licenses are present
+    const cluster = instantiateClient(
+      config.ui.elasticsearch,
+      this.log,
+      core.elasticsearch.createClient
+    );
```

Comment:
As monitoring is not migrated to the new client yet (due to this issue), I instantiated the new client at the only place we currently need it. I will let the owners refactor that when migrating the rest of the plugin